packages/cli/src/util/build/import-builders.ts

```diff
@@ -201,15 +205,46 @@ async function installBuilders(
     if (err.code !== 'EEXIST') throw err;
   }
 
-  output.debug(`Installing Builders: ${Array.from(buildersToAdd).join(', ')}`);
-  await spawnAsync('yarn', ['add', '@vercel/build-utils', ...buildersToAdd], {
-    cwd: buildersDir,
-  });
+  output.log(`Installing Builders: ${Array.from(buildersToAdd).join(', ')}`);
+  try {
+    await spawnAsync('yarn', ['add', '@vercel/build-utils', ...buildersToAdd], {
+      cwd: buildersDir,
+      stdio: 'pipe',
+    });
+  } catch (err: unknown) {
+    if (isError(err)) {
+      if (isErrnoException(err) && err.code === 'ENOENT') {
+        // `yarn` is not installed
```

Comment:
Should we use npm instead since that is more likely to be available on the user's system?

Reply:
Ya possibly. I don't remember why I used `yarn` here originally. I'll see how easy it is to change.

Reply:
Probably because thats how the build container did it. 

But we can guarantee yarn in the build container but we can't guarantee on the user's local machine.

Reply:
It would be neat if it detected a `yarn.lock` or `pnpm-lock.yaml` and spawned the corresponding package manager, otherwise fallback to `npm`.

Reply:
In this case it's related to the installation of Builders which happens in `.vercel/builders` (so we control it and there is no lockfile). But in general, ya I agree, we should have all of this logic in a `addDependencies()` function in `build-utils` similar to how we have the `installDependencies()` function.